Src/AutoFixture/Fare/Xeger.cs

```diff
@@ -18,15 +18,18 @@
  */
 
 using System;
+using System.Diagnostics.CodeAnalysis;
 using System.Text;
 
-namespace AutoFixture.DataAnnotations
+namespace Fare
 {
     /// <summary>
     /// An object that will generate text from a regular expression. In a way, 
     /// it's the opposite of a regular expression matcher: an instance of this class
     /// will produce text that is guaranteed to match the regular expression passed in.
     /// </summary>
+    [SuppressMessage("Microsoft.Naming", "CA1724:TypeNamesShouldNotMatchNamespaces", 
+        Justification = "This namespace contains everything specific to this module implementation, so call it the same.")]
```

Comment:
Note sure why this suppression is needed.

Reply:
Nice catch 🏇 That was a left-over from the previous implementation and now it's indeed not required more. Removed it.